docs/cluster.md

```diff
@@ -37,20 +37,19 @@ We recommend you stick to the outlined examples for deploying cluster environmen
 All the cluster commands use both the `--tag` and `--provider` options, which specify the tag for
 the cluster images as well as the target for the cluster you are performing operations on.
 
-
-
 #### Creating a cluster on your local system
 
 This is the default mode.
 
 * To create a cluster locally:
 ```
-$ amp cluster create --provider local
+$ amp cluster create --tag [AMP version]
 ...
 Cluster status: healthy
 ...
 ```
-This will deploy the `AMP` stack on your local docker engine. This AMP deployment only
+This will deploy the `AMP` stack on your local docker engine using the image version specified with the `--tag` flag.
+If you are using the unstable, development version of `AMP`, you will not need to provide this option. This AMP deployment only
```

Comment:
s/unstable, //
and the tag is not mandatory either with the released version of AMP, it's only useful when one wants to deploy the cluster in a different version than the CLI.